source/extensions/filters/http/grpc_http1_bridge/config.cc

```diff
@@ -10,10 +10,11 @@ namespace HttpFilters {
 namespace GrpcHttp1Bridge {
 
 Http::FilterFactoryCb GrpcHttp1BridgeFilterConfig::createFilterFactoryFromProtoTyped(
-    const envoy::extensions::filters::http::grpc_http1_bridge::v3::Config&, const std::string&,
-    Server::Configuration::FactoryContext& factory_context) {
-  return [&factory_context](Http::FilterChainFactoryCallbacks& callbacks) {
-    callbacks.addStreamFilter(std::make_shared<Http1BridgeFilter>(factory_context.grpcContext()));
+    const envoy::extensions::filters::http::grpc_http1_bridge::v3::Config& proto_config,
+    const std::string&, Server::Configuration::FactoryContext& factory_context) {
+  return [&factory_context, proto_config](Http::FilterChainFactoryCallbacks& callbacks) {
+    callbacks.addStreamFilter(std::make_shared<Http1BridgeFilter>(
+        factory_context.grpcContext(), proto_config.upgrade_protobuf_to_grpc()));
```

Comment:
The 2nd parameter should probably be `proto_config`, just in case new fields will be added in the future.

Reply:
I implemented your suggestion, but noticed that some filters I looked at seem to pass the attributes directly instead of the protobufs. Let me know what you think, I don't know how strongly we'd prefer to stick to that convention instead, this was me essentially cargo culting a bit :shrug: